recipe/meta.yaml

```diff
@@ -13,7 +13,8 @@ source:
 
 build:
   # Skip py35 since py36 is built and uses same vc14
-  number: 1002
+  number: 1002             # [not win]
+  number: 1003             # [win]
```

Comment:
We usually try to keep the build no. consistent across platforms. Is there anything preventing us from uploading build 1003 also to [not win]?

Reply:
I think it would be fine to bump the build number to 1003 across all platform, except that this fix only addresses Windows.

Reply:
I think we should go with 1003 IMHO.  We should bump build number here in common